src/PhoneNumber/PhoneNumber.js

```diff
@@ -84,73 +57,29 @@ class PhoneNumber extends Component {
   };
 
   handleChange = e => {
-    const { value: currentNumber, selectionStart: caretPosition } = e.target;
-    const { phonePattern, value: selectedCountry } = this.getCountryFrom(
-      currentNumber
-    );
+    const { value: currentNumber } = e.target;
+    const { dialingCode } = this.state;
 
-    let formattedNumber = currentNumber.replace(/(?!^\+)\D/gm, '');
+    const phoneNumber = currentNumber.replace(/[^\d]/gm, '');
 
-    if (phonePattern) {
-      formattedNumber = this.formatNumber(formattedNumber, phonePattern);
-    }
+    const formattedNumber = dialingCode
```

Comment:
Is it possible to not have a dialing code?

Reply:
I think the component shouldn't be a restriction. In normal use it is usually filled in beforehand